Locales/deDE.lua

```diff
@@ -1,12 +1,12 @@
 local L = LibStub("AceLocale-3.0"):NewLocale("Big Wigs", "deDE")
 if not L then return end
 
---L.getNewRelease = "Your Big Wigs is old (/bwv) but you can easily update it using the Curse Client. Alternatively, you can update manually from curse.com or wowinterface.com."
---L.warnTwoReleases = "Your Big Wigs is 2 releases out of date! Your version may have bugs, missing features, or completely incorrect timers. It is strongly recommended you update."
---L.warnSeveralReleases = "|cffff0000Your Big Wigs is several releases out of date!! We HIGHLY recommend you update to prevent syncing issues with other players!|r"
+L.getNewRelease = "Dein Big Wigs ist veraltet (/bwv), aber du kannst es mit Hilfe des Curse Client einfach aktualisieren. Alternativ kannst du es auch manuell von curse.com oder wowinterface.com herunterladen und manuell aktualisieren."
+L.warnTwoReleases = "Dein Big Wigs ist 2 Versionen älter als die neueste Version! Deine Version könnte Fehler, fehlende Funktionen oder völlig falsche Zeitmesser beinhalten. Es wird dringend empfohlen, Big Wigs zu aktualisieren."
```

Comment:
Please don't translate `timer` with `Zeitmesser` that's just super weird. Actually I would just not translate it at all.